src/main/java/org/jetbrains/bsp/bazel/server/util/ParsingUtils.java

```diff
@@ -23,7 +25,6 @@
 public class ParsingUtils {
```

Comment:
I would consider moving this class or part of it closer to the usage - most of the logic is strongly related to the implementation details other classes.

Reply:
You're right, it really makes sense to disassemble this class a bit now. It was created when many of the classes were large and mixed in one place and I desperately needed to extract some methods somewhere. But now that the structure is more clear we should definitely move some of the methods (those that are not very generic and have logic bound to specific components), either back to the original class or to a separate helper class in its package. It's a bit of work for later though, maybe I'll create an issue out of this one, too?

Reply:
coool